src/dialogs/more-info/controls/more-info-siren.ts

```diff
@@ -0,0 +1,130 @@
+import "@material/mwc-button/mwc-button";
+import { css, CSSResultGroup, html, LitElement, TemplateResult } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { stopPropagation } from "../../../common/dom/stop_propagation";
+import { supportsFeature } from "../../../common/entity/supports-feature";
+import "../../../components/ha-duration-input";
+import { UNAVAILABLE } from "../../../data/entity";
+import { SirenEntity, SirenEntityFeature } from "../../../data/siren";
+import { HomeAssistant } from "../../../types";
+
+const DEFAULT_TONE_KEY = "_";
+
+@customElement("more-info-siren")
+class MoreInfoSiren extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj?: SirenEntity;
+
+  @state() _selectedTone?: string;
+
+  private _toneChanged(ev) {
+    const newVal = ev.target.value;
+
+    if (newVal === DEFAULT_TONE_KEY) {
+      delete this._selectedTone;
+      return;
+    }
+
+    this._selectedTone = newVal;
+
+    if (this.stateObj?.state === "on") {
+      this._turnOn();
+    }
```

Comment:
It is not possible to set a tone without turning it on? So selecting a tone when the siren is off doesn't do anything? It is not saved?

Reply:
No, it's like the brightness for light. I pushed a quick PR in draft mode so we can discuss on it during frontend meeting.